### PR TITLE
CLOUDP-68114: Add CAS field for security certs

### PR DESCRIPTION
**Pull request by robcarlan-mlab**
## Proposed changes

_Jira ticket:_ CLOUDP-68114

Closes #[issue number]

## Checklist

- [] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [] I have added tests that prove my fix is effective or that my feature works
- [] I have added any necessary documentation (if appropriate)
- [] I have run `make fmt` and formatted my code

## Further comments

Just outputs the plain certificate:

```
./bin/mongocli atlas security customerCerts create --casFile ~/.ssh/bad_pem.pem
Certificate successfully created.
```

```
./bin/mongocli atlas security customerCerts describe
-----BEGIN CERTIFICATE-----
....
bSQR/NrJDA==
-----END CERTIFICATE-----

```

### CODE REVIEW

File: internal/cli/atlas/security/customercerts/create.go [hunk 1]
```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const createTemplate = `CAS
+{{.Cas}}
+`
+
```
Comment (colm-quinn) [inline internal/cli/atlas/security/customercerts/create.go:32]: [q] What did this look like without the template?

I'm looking at the output fro `gpg` and wondering should it just be the key, so you can then pipe the output to file and use it directly somewhere. Right now you'd need to strip the `CAS` line at the start.

Comment (gssbzn) [reply internal/cli/atlas/security/customercerts/create.go:32]: @robcarlan-mlab I'm with @colm-quinn in that I feel like the "human" output could be just straight up the certificate, no title, but that for the describe, for the create I think we should follow other commands with just a "success message"

@colm-quinn for context in this command (create) the user is providing the certificate so returning an answer with the same certificate the user provided is kind of a meh for me

Comment (colm-quinn) [reply internal/cli/atlas/security/customercerts/create.go:32]: > @colm-quinn for context in this command (create) the user is providing the certificate so returning an answer with the same certificate the user provided is kind of a meh for me

Got it and yes agreed.

Comment (robcarlan-mlab) [reply internal/cli/atlas/security/customercerts/create.go:32]: Yeah that makes sense to me too 👍

**Pull request merged**